package.json

```diff
@@ -88,7 +88,7 @@
     "@libp2p/tcp": "^10.0.7",
     "@libp2p/websockets": "^9.0.6",
     "@multiformats/multiaddr": "^12.3.1",
-    "@oceanprotocol/lib": "^3.3.3",
+    "@oceanprotocol/lib": "^4.0.0-next.1",
```

Comment:
We shouldn't be using this `4.0.0-next.1`, Alex said he was going to remove it... We should be using npm link to test this.